main.scss

```diff
@@ -8,6 +8,18 @@ $sans-serif: oFontsGetFontFamilyWithFallbacks(MetricWeb);
 
 // Add this class to your main page wrapper
 
+.kat-root {
+	min-height: 100%;
+	position: relative;
+	padding-bottom: 250px;
```

Comment:
Would be nice if the padding was in `em` rather than `px` to cater for users with a larger font size, as that could affect the height of the footer.